src/alert/Alert.styles.js

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+
+import Icon from '../icon/Icon';
+
+import { colors, sizes } from './theme';
+
+const StyledAlert = styled.div`
+  padding: ${({ size }) => sizes[size].padding};
+  border-radius: 3px;
+  border: none;
+  margin-bottom: 1px;
```

Comment:
Looks good overall, but I think we should make the margin bottom be the same as Message so its consistent across the components